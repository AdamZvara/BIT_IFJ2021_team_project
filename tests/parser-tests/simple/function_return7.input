require "ifj21"

global bar : function()

function foo()
	write("TOTO vypis")
	return
	write("Honza Cernocky je legenda\n")
end

function bar()
	foo()

end

bar()
