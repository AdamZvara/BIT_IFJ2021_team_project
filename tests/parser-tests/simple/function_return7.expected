TOTO vypisHonza Cernocky je legenda
